cmd/subctl/deploybroker.go

```diff
@@ -24,13 +24,13 @@ import (
 
 	"github.com/spf13/cobra"
 	"github.com/submariner-io/admiral/pkg/stringset"
-	"github.com/submariner-io/submariner-operator/internal/cli"
-	"github.com/submariner-io/submariner-operator/internal/component"
-	"github.com/submariner-io/submariner-operator/internal/constants"
-	"github.com/submariner-io/submariner-operator/internal/exit"
-	"github.com/submariner-io/submariner-operator/pkg/broker"
-	"github.com/submariner-io/submariner-operator/pkg/client"
-	"github.com/submariner-io/submariner-operator/pkg/deploy"
+	"github.com/submariner-io/subctl/internal/cli"
+	"github.com/submariner-io/subctl/internal/component"
+	"github.com/submariner-io/subctl/internal/constants"
+	"github.com/submariner-io/subctl/internal/exit"
+	"github.com/submariner-io/subctl/pkg/broker"
+	"github.com/submariner-io/subctl/pkg/client"
```

Comment:
Lets reuse this from operator itself.

Reply:
Seems commit [b34d651](https://github.com/submariner-io/subctl/pull/5/commits/b34d651dc745478bf1650ad45f1a4a8019b2c2d8) was pushed while I was reviewing.

Reply:
That has the client `Producer` which is only used by `subctl`. The controllers use the `sigs.k8s.io/controller-runtime/pkg/client` interface.